solr/conf/schema.xml

```diff
@@ -177,11 +160,9 @@
       <analyzer>
         <!-- <tokenizer class="solr.WhitespaceTokenizerFactory"/> -->
         <tokenizer class="solr.StandardTokenizerFactory"/>
-        <filter class="solr.ICUFoldingFilterFactory" />
```

Comment:
This might be the issue: https://issues.apache.org/jira/browse/SOLR-6188
I'll try using the fully qualified class name.